src/main/java/persistence/sql/clause/WhereConditionalClause.java

```diff
@@ -0,0 +1,54 @@
+package persistence.sql.clause;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public record WhereConditionalClause(String column, String value, String operator) implements ConditionalClause {
+
+    public static WhereExpression builder() {
+        return new WhereExpression();
+    }
+
+    public static class WhereExpression {
+        private String column;
+        private String value;
+        private String operator;
+
+        public WhereExpression column(String column) {
+            this.column = column;
+            return this;
+        }
+
+        public WhereConditionalClause eq(String value) {
+            this.value = value;
+            this.operator = "=";
+            return build();
+        }
+
+        public WhereConditionalClause in(List<Object> values) {
+            this.value = values.stream()
+                    .map(Clause::toColumnValue)
+                    .collect(Collectors.joining(" , "));
+            this.operator = "IN";
+            return build();
+        }
+
+        public WhereConditionalClause neq(String value) {
+            this.value = value;
+            this.operator = "!=";
+            return build();
+        }
+
+        public WhereConditionalClause notIn(List<Object> values) {
+            this.value = values.stream()
+                    .map(Clause::toColumnValue)
+                    .collect(Collectors.joining(" , "));
+            this.operator = "NOT IN";
+            return build();
+        }
```

Comment:
```suggestion
        public WhereConditionalClause in(List<Object> values) {
            this.value = values.stream()
                    .map(Clause::toColumnValue)
                    .collect(Collectors.joining(" , "));
            this.operator = "IN";
            return build();
        }

        public WhereConditionalClause neq(String value) {
            this.value = value;
            this.operator = "!=";
            return build();
        }

        public WhereConditionalClause notIn(List<Object> values) {
            this.value = values.stream()
                    .map(Clause::toColumnValue)
                    .collect(Collectors.joining(" , "));
            this.operator = "NOT IN";
            return build();
        }
```
새로운 패턴으로 `operator`를 적용하기 쉬워진 포인트가 존재하는군요~!
하지만, 테스트코드가 있었으면 더 좋았을것 같은데 아직은 사용되고 있지 않네요 ^^;;